gems/pending/Gemfile

```diff
@@ -22,7 +22,7 @@ gem "hawkular-client",         "=2.7.0",            :require => false
 gem "httpclient",              "~>2.7.1",           :require => false
 gem "image-inspector-client",  "~>1.0.3",           :require => false
 gem "iniparse",                                     :require => false
-gem "kubeclient",              "=2.3.0",            :require => false
+gem "kubeclient",              "~>2.4.0",           :require => false
```

Comment:
because it's a backport?
we're already open to minor-version bumps in many gems like `image-inspector-client` above.
IMO if we care about controlling gem updates in stable branches, we should version control Gemfile.lock, not have replace `~>`s with `=`s.
But I'm fine either way, somebody please decide :)